br/pkg/trace/main_test.go

```diff
@@ -0,0 +1,13 @@
+package trace
```

Comment:
Should have a license header, you can convey from https://github.com/pingcap/tidb/blob/cddb584acd83ff87797aae766c237fd8a056b260/expression/aggregation/main_test.go#L1-L13

Reply:
I'm sorry, I will fix it.